Gemfile.lock

```diff
@@ -412,7 +403,7 @@ GEM
       net-ssh (>= 2.8.0)
     sshkit-interactive (0.3.0)
       sshkit (~> 1.12)
-    thor (0.20.3)
+    thor (1.0.1)
```

Comment:
### [thor](http://whatisthor.com/)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 0.20.3 → 1.0.1

[_[change-log](https://github.com/erikhuda/thor/blob/master/CHANGELOG.md), [source-code](https://github.com/erikhuda/thor/tree/v1.0.1)_]





<details>
<summary>Commits</summary>

A change of **114** commits. See the full changes on [the compare page](https://github.com/erikhuda/thor/compare/v0.20.3...v1.0.1).

These are the first 10 commits:
- (b2b1e7a) [Add Thor.deprecation_warning](https://github.com/erikhuda/thor/commit/b2b1e7aedf721ff069292f909acccb01d9a1d858)
- (7d199c0) [Always check default type. Deprecate incompatible types with…](https://github.com/erikhuda/thor/commit/7d199c0367997939857a3c3d96209515239989e3)
- (c7e38fd) [Add Thor.deprecation_warning](https://github.com/erikhuda/thor/commit/c7e38fd6d2fb41fdc7039644ae91f97a0ef64878)
- (cc8e8fd) [Deprecate relying on default exit_on_failure? [#621]](https://github.com/erikhuda/thor/commit/cc8e8fd59fd5a7c2909e80bfa1ff90dee1193d4b)
- (562629b) [Fix typo [ci skip]](https://github.com/erikhuda/thor/commit/562629bd0c1152c7ce8f341176c346134613ea3c)
- (1ec3eff) [Merge pull request #638 from yujideveloper/fix/typo](https://github.com/erikhuda/thor/commit/1ec3effc78f2a849924c2fb7835adcfe60985020)
- (371f2a3) [Fix erikhuda/thor#587](https://github.com/erikhuda/thor/commit/371f2a3c549e0bc57ac1f53d0050c340d71fb5e9)
- (4344761) [Merge pull request #639 from hugocorbucci/master](https://github.com/erikhuda/thor/commit/4344761bb555897f31ae7575ab7a60fe908893c8)
- (8c57f1d) [Do not require did_you_mean](https://github.com/erikhuda/thor/commit/8c57f1dee1d6cb9d0071c793eae05278ece4d8c3)
- (d3762d0) [Merge pull request #640 from quixoten/patch-1](https://github.com/erikhuda/thor/commit/d3762d0b89a80da488285d640b052f5276fac6bb)


</details>